mlrun/datastore/datastore_profile.py

```diff
@@ -344,24 +344,24 @@ class DatastoreProfileAzureBlob(DatastoreProfile):
     client_secret: typing.Optional[str] = None
     sas_token: typing.Optional[str] = None
     credential: typing.Optional[str] = None
-    bucket: typing.Optional[str] = None
+    container: typing.Optional[str] = None
 
-    @pydantic.validator("bucket")
-    def check_bucket(cls, v):
+    @pydantic.validator("container")
+    def check_container(cls, v):
```

Comment:
Need to add `@classmethod` or change `cls` -> `self`.
(all occurrences)

Reply:
@jond01

Reply:
The relevant rules are:
* N804: https://docs.astral.sh/ruff/rules/invalid-first-argument-name-for-class-method/
* N805: https://docs.astral.sh/ruff/rules/invalid-first-argument-name-for-method/

They are a part of the pep8-naming rules (`N`), which are not selected in mlrun. I encourage you to add it 🙂